src/emmy/expression/render.cljc

```diff
@@ -39,9 +26,12 @@
   simplifier negates by wrapping with `(* -1 ...)`. For rendering, we prefer to
   use a unary minus."}
   rewrite-negation
+
   (R/ruleset
-   (* -1 ?x) => (u- ?x)
-   (* -1 ??x) => (u- (* ??x))))
+   (* (? _ #{-1 -1.0}) ?x)               => (u- ?x)
```

Comment:
I hate to add one more literal but do we need `1N` here too?

Reply:
we're ok:
```clojure
user=> (#{1 1.0} 1N)
1
```